### DISCUSSION TITLE
How do I set a fixed width for each column in TableVirtuoso?

### DISCUSSION THREAD

**Discussion by mapringg** [petyosi/react-virtuoso#622] [Q&A]
The default behavior changes the column width while I'm scrolling the table which is not ideal.

I would like each column to always maintain 150px width.

I tried adjusting itemSize, but that didn't have any effect.

**Comment by gadicc** [comment]
As a workaround, if you use [fixed-headers](https://virtuoso.dev/table-fixed-headers/) and specify the width in those cells, it works.

*Edit:* if the header cells are empty and have no `paddingTop` or `paddingBottom` (but otherwise match padding and other styles of the rest of your table), it should appear invisible.  (On the other hand, `display: none` results in the same problem).
